### DISCUSSION TITLE
A new benchmark for 'in-the-wild' distributional shift

### DISCUSSION THREAD

**Discussion by Haoxiang-Wang** [p-lambda/wilds#76] [General]
Hi WILDS team,

I just notice another new benchmark for distribution shift (containing 3 new datasets), which you may find interesting.

arXiv: https://arxiv.org/abs/2107.07455
GitHub: https://github.com/yandex-research/shifts

**Comment by kohpangwei** [comment]
Hi Haoxiang,

Thank you for pointing this out to us! Their datasets are interesting and we're glad that there are more works coming out on real-world distribution shifts. 🙂
